Makefile

```diff
@@ -1,13 +1,7 @@
 REGISTRY ?= quay.io
 DEFAULT_TAG = latest
-
 DOCKERFILE := Dockerfile
-ifeq ($(TARGET),rhel)
```

Comment:
This will also require a change in the saas analytics repo.

Reply:
@prashbnair We have to move this deployment to APP_INTERFACE any ways, that's next task I am following after this PR.